index.html

```diff
@@ -58,8 +58,10 @@ <h2>Chingu Raptors</h2>
   </section>
   <section id="dino-index">
     <h2>Dinosaur Index</h2>
-    <ul id="dino-list"></ul>
+    <ul id="dino-list"></ul>]
```

Comment:
Our `#dino-list` container is our `root` element for displaying dinosaurs.
If you don't style this element then none of the dinosaurs within will accept the styles.

The reason you can see the responsive design is because @JOSIK95 applied a flexbox container to this container using CSS yesterday.